spec/models/aws_engine_spec.rb

```diff
@@ -2,6 +2,81 @@
 require 'spec_helper'
 require 'digest/md5'
 
+module Aws
+  module SQS
+    class DummyClient
+      def send_message(params = {}, options = {})
+        nil
+      end
+      def receive_message(params = {})
+        []
+      end
+      def delete_message(params = {})
```

Comment:
__[reek]__
* [UnusedParameters] has unused parameter 'params' [:ref](https://github.com/troessner/reek/blob/master/docs/Unused-Parameters.md)
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)